NASA-IMPACT/veda-ui
# Details styling

**Related Ticket:** _{link related ticket here}_

### Description of Changes
This PR contains changes to show details of the queries and it's parts that matches the responses by the agents.

### Validation / Testing
- Check to see if words/phrases are highlighted
- Hover over highlights provide details of what they were matched to.
